sherlock/internal/models/changeset_plan_apply.go

```diff
@@ -204,18 +205,18 @@ func changesetPostApplyActions(db *gorm.DB, appliedChangesets []Changeset) {
 				// here, so we should really test it.)
 				//
 				//// 3. Report to Pact
-				//if environment.PactIdentifier != nil {
-				//	var chart Chart
-				//	if errToSwallow := db.
-				//		Take(&chart, appliedChangeset.ChartRelease.ChartID).Error; errToSwallow != nil {
-				//		go slack.ReportError(context.Background(), fmt.Sprintf("unable to load chart %d after applying changeset %d", appliedChangeset.ChartRelease.ChartID, appliedChangeset.ID), errToSwallow)
-				//	} else if chart.PactParticipant != nil && *chart.PactParticipant && appliedChangeset.To.AppVersionExact != nil {
-				//		go pactbroker.RecordDeployment(
-				//			chart.Name,
-				//			*appliedChangeset.To.AppVersionExact,
-				//			*environment.PactIdentifier)
-				//	}
-				//}
+				if environment.PactIdentifier != nil {
+					var chart Chart
```

Comment:
This is a lot of pact specific logic in this method, 
some wrapper method like pactbroker.ReportChangesetBump (or something) should just get called here and forgotten about

Reply:
Let's do this in another ticket as the focus of this ticket is to get the pactbroker test to prove there is no panic when a changeset is applied.